internal/application/workload.go

```diff
@@ -183,9 +184,19 @@ func (a *Workload) Get(ctx context.Context) (*models.AppDeployment, error) {
 		// Initialize various pieces from the first pod ...
 		createdAt = podList[0].ObjectMeta.CreationTimestamp.Time
 		stageID = podList[0].ObjectMeta.Labels["epinio.io/stage-id"]
-		username = podList[0].ObjectMeta.Labels["app.kubernetes.io/created-by"]
 		controllerName = podList[0].ObjectMeta.Labels["epinio.io/app-container"]
 
+		userID := podList[0].ObjectMeta.Labels["app.kubernetes.io/created-by"]
+		authService, err := auth.NewAuthServiceFromContext(ctx)
+		if err != nil {
+			return nil, err
+		}
+		user, err := authService.GetUserByID(ctx, userID)
```

Comment:
Previously, it was fine if the user was no longer there. I think it still is. E.g. A user creates a resource, the operators delete this user, someone tries to fetch the resource. There is no reason why this should fail. Maybe fallback to `[Deleted user]` or something?